controllers/app.controller.js

```diff
@@ -0,0 +1,10 @@
+const { selectTopics } = require("../modules/app.module");
+
+exports.getTopics = (req, res, next) => {
+  selectTopics()
+    .then((topics) => {
+      res.status(200).send({ topics });
```

Comment:
Nice function naming, great that you're sending the response in an object on an appropriate key 🔑